engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2246,6 +2306,67 @@ public LogicalOperatorInfo getLogicalOperatorInfo(String operatorName)
     return fillLogicalOperatorInfo(operatorMeta);
   }
 
+  public LogicalModuleInfo getLogicalModuleInfo(String moduleName, boolean flatten, LogicalPlan dag, String ancestry)
```

Comment:
This is the plan for the future, Adding attribute on module to specify how operators needs to be partitioned. like make a copy of module dag in case of parallel partition. Currently module writer has to specify correct partitioner for individual operators he wants to partitioned. Currently module is implemented just as a template.